src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java

```diff
@@ -553,19 +554,16 @@ public ListBoxModel doFillRegionItems(@QueryParameter final String credentialsId
                                               @QueryParameter final String region)
                 throws IOException, ServletException {
             final List<Region> regionList;
-
             try {
-                final AmazonEC2 client = connect(credentialsId, null);
-                final DescribeRegionsResult regions=client.describeRegions();
-                regionList=regions.getRegions();
+                regionList=RegionUtils.getRegionsForService(ServiceAbbreviations.EC2);
```

Comment:
This approach is really good, however it's using static information from ```aws-java-sdk-core``` lib, so if you region created and lib was not updated you will not see it here, for example with current version you will not see ```eu-west-3``` Paris region, so somehow we should always update lib, do we rally need that?